templates/openshift-prune-cronjob.yml.j2

```diff
@@ -12,13 +12,13 @@ spec:
           - name: token
             secret:
               defaultMode: 420
-              secretName: {{ osbs_prune_secret }}
+              secretName: "{{ osbs_prune_secret }}"
           containers:
           - name: build-pruner
-            image: {{ osbs_prune_image }}
-            {-% if osbs_prune_commands %}
-            command: {{ osbs_prune_commands }}
-            {% endif -%}
+            image: "{{ osbs_prune_image }}"
+
+            {% if osbs_prune_commands %}command: {{ osbs_prune_commands }}{% endif %}
```

Comment:
Does `{{ osbs_prune_commands }}` also need double-quotes here?

Reply:
No, its actually a list of strings - this is the recommended way to pass cmdline arguments in openshift yamls

Reply:
I'm wondering if setting command to null has the same effect as omitting it.